### ISSUE TITLE
DOC: ecosystem.rst: Vaex

### ISSUE THREAD

**Issue by westurner** [pandas-dev/pandas#20335]
Is it appropriate to add vaex to ecosystem.rst? What should the heading link and description be?

- Src: https://github.com/maartenbreddels/vaex
- Docs: https://docs.vaex.io/

>  Vaex is a python library for Out-of-Core DataFrames (similar to Pandas), to visualize and explore big tabular datasets. It can calculate statistics such as mean, sum, count, standard deviation etc, on an N-dimensional grid up to a billion (109) objects/rows per second. Visualization is done using histograms, density plots and 3d volume rendering, allowing interactive exploration of big data. Vaex uses memory mapping, zero memory copy policy and lazy computations for best performance (no memory wasted).

- https://github.com/maartenbreddels/vaex/blob/master/docs/source/getting_data_in_vaex.rst
  - vaex.from_pandas
  - vaex.to_pandas_df

https://github.com/pandas-dev/pandas/blob/master/doc/source/ecosystem.rst

**Comment by TomAugspurger** [comment]
👍  @maartenbreddels is that a good description for inclusion in http://pandas.pydata.org/pandas-docs/stable/ecosystem.html?

Side-note, we should maybe rephrase the opening sentence

> Increasingly, packages are being built on top of pandas to address specific needs in data preparation, analysis and visualization.

That implies (... states) that all these are built on top of pandas. Some are, but others just have nice integration to / from pandas.

**Comment by maartenbreddels** [comment]
👍 thanks, that looks good indeed, maybe replace (109) by $10^9$ is latex is supported, otherwise 10**9 for the Python readers.

**Comment by westurner** [comment]
```rst
10\ :sup:`9`
```
